### PR TITLE
[SMALLFIX]Add java doc for CapacityTest

### PR DESCRIPTION
**Pull request by Popeye13**

### CODE REVIEW

File: core/common/src/test/java/alluxio/wire/CapacityTest.java [hunk 1]
```diff
@@ -21,6 +21,11 @@
 
 public class CapacityTest {
 
+  /**
+   * Test to convert between a Capacity type and a json type.
+   *
+   * @throws Exception if an error occurs during convert between Capacity type and json type
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/wire/CapacityTest.java]: convert -> conversion

File: core/common/src/test/java/alluxio/wire/CapacityTest.java [hunk 2]
```diff
@@ -34,12 +39,23 @@ public void equals() {
     CommonTestUtils.testEquals(AlluxioMasterInfo.class);
   }
 
+  /**
+   * Check if the two Capacity object are equal.
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/wire/CapacityTest.java]: Check -> Checks, object -> objects

File: core/common/src/test/java/alluxio/wire/CapacityTest.java [hunk 3]
```diff
@@ -34,12 +39,23 @@ public void equals() {
...
+   * @param b the second Capacity object to be checked
+   */
   private void checkEquality(Capacity a, Capacity b) {
     Assert.assertEquals(a.getTotal(), b.getTotal());
     Assert.assertEquals(a.getUsed(), b.getUsed());
     Assert.assertEquals(a, b);
   }
 
+  /**
+   * Randomly create a Capacity object.
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/wire/CapacityTest.java]: create -> creates

Comment (Popeye13) [inline core/common/src/test/java/alluxio/wire/CapacityTest.java]: @yupeng9 Thanks for your review, updated

**Comment by apc999** [comment]
@yupeng9  do you have more comments?

**Comment by yupeng9** [comment]
@Popeye13 can you resolve the conflicts?

**Comment by Popeye13** [comment]
@yupeng9 Thanks for your review, the conflicts resolved.

**Comment by yupeng9** [comment]
Thanks for the contribution.

**Pull request merged**